CMakeLists.txt

```diff
@@ -3682,6 +3709,243 @@ if(gRPC_INSTALL)
   )
 endif()
 
+if(gRPC_BUILD_TESTS)
+
+add_library(uv_linux
```

Comment:
this seems wrong, you should be using the libuv targets provided by libuv's CmakeLists.txt, not constructing the libraries here.

Should you list the libuv targets here: https://github.com/grpc/grpc/blob/84cf6660147f91f616e886757cbaf7ccedbe01c5/templates/CMakeLists.txt.template#L458?

Reply:
Fixed as suggested.